felix/fv/named_ports_test.go

```diff
@@ -216,17 +215,6 @@ func describeNamedPortTests(testSourcePorts bool, protocol string) {
 		applyAtOthers
 	)
 
-	bpfEnabled := false
```

Comment:
What about UDP?  Might still need the clean in that case

Reply:
how could that differ from what iptables do? There is no sense of a new connection :thinking: